src/renderer/components/editorWithTabs/tabs.vue

```diff
@@ -82,13 +82,15 @@ export default {
       const newLeft = Math.max(0, Math.min(tabs.scrollLeft + delta, tabs.scrollWidth))
       tabs.scrollLeft = newLeft
     },
-    closeTab (tab) {
-      if (tab.id) {
+    closeTab (tabId) {
+      const tab = this.tabs.find(f => f.id === tabId)
```

Comment:
Do not repeat yourself.

Reply:
Well... to be honest, I don't understand why @fxha didn't like to pass `tab` object itself to menu item.
@fxha Can I know your opinion?

Reply:
I think personal preferences but also that we don't spread object all over. All newer API/functions should use just id's and older need refactoring.  Electrons menu items are main process component and if we bind object to it, the values are copied between main and renderer process which is not necessary.

You can simply forward the tab id to the vue function if possible like:

```js
// file 1
this.$store.dispatch('CLOSE_OTHER_TABS', tabId)

// store/editor.js

CLOSE_OTHER_TABS ({ state, dispatch }, tabId) {
    const { tabs } = state
    tabs.filter(f => f.id !== tabId).forEach(tab => {
// ...
}
````